### DISCUSSION TITLE
Trying to edit workbench.html

### DISCUSSION THREAD

**Discussion by Sharpz7** [coder/code-server#5238] [Q&A]
Hi,

I am trying to add Fira Code Manually using Dockerfiles. Right now, I am just testing by directly editing workbench.html

https://files.mcaq.me/cbcx1.png
I added these changes.

And get this result:
https://files.mcaq.me/94hb2.png

But when I remove 'unsafe-inline', (by removing the last letter) I then get:
https://files.mcaq.me/r3118.png

What seems to be happening is the "fonts.googleapis.com" is being ignored when "unsafe-inline" is present

Any ideas?

Thanks

**Comment by lg-nineoneninethreenine** [comment]
I don't know if you should edit workbench.html, it could be workbench.js, right?

**Reply by Sharpz7** [reply]
Why would it be that? I need to set a meta tag.

**Reply by lg-nineoneninethreenine** [reply]
Um idk, but @code-asher can tell...

**Comment by Sharpz7** [comment]
An update: 

https://files.mcaq.me/3072r.png

I think it is this iframe tag causing the problems.

**Reply by jsjoeio** [reply]
Did you try this? https://github.com/coder/code-server/issues/1374#issuecomment-1143659522

**Reply by Sharpz7** [reply]
Yeah thanks for getting back to me. By using the css file you compile the fonts into the css before the content security policy is implemented ;)

Would be cool to see this as some sort of ENV var in the future.

**Reply by jsjoeio** [reply]
Hmm...if you want to write an RFC/proposal for that, we could consider it!

**Reply by Sharpz7** [reply]
Okay! Would that be done as an issue feature request?

**Reply by jsjoeio** [reply]
Yeah sure! Once you make it, I can update labels/etc.
